packages/grafana-ui/src/components/Forms/Select/SelectBase.tsx

```diff
@@ -1,12 +1,14 @@
 import React from 'react';
 import { SelectableValue, deprecationWarning } from '@grafana/data';
 // @ts-ignore
-import { default as ReactSelect, Creatable } from '@torkelo/react-select';
+import { default as ReactSelect } from '@torkelo/react-select';
 // @ts-ignore
-import { default as ReactAsyncSelect } from '@torkelo/react-select/lib/Async';
+import Creatable from '@torkelo/react-select/creatable';
```

Comment:
merge with import above?

Reply:
https://github.com/JedWatson/react-select/issues/3585

Apparently it is not supported.